metaflow/plugins/aws/eks/kubernetes.py

```diff
@@ -358,9 +358,7 @@ def _print_available(tail, stream, should_persist=False):
                 "%s. This could be a transient error. "
                 "Use @retry to retry." % msg
             )
-        elif not self._job.is_done:
-            # Kill the job if it is still running by throwing an exception.
-            raise KubernetesKilledException("Task failed!")
+
```

Comment:
Removed since it is no longer possible for `RunningJob` to be not "running" anymore and not "done" at the same time.